content/get-started/workshop/04_sharing_app.md

```diff
@@ -52,6 +52,10 @@ In the following image, you can see an example Docker command from Docker Hub. T
 
     To fix this, you need to tag your existing image you've built to give it another name.
 
+   > [!NOTE]
+   >
+   > If you encounter the following error: `denied: requested access to the resource is denied`, you can retry with `$ docker login -u "myusername" -p "mypassword" docker.io`.
+
 2. Sign in to Docker Hub using the command `docker login -u YOUR-USER-NAME`.
```

Comment:
Perhaps we should remove the `-u` as well, as that skips the OAuth workflow. Logging in without the `-u` option will ask the user for their username, so we may need to call that out (type `docker login` and enter your username .. )